CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 ### Under the hood
 - Added 3 more adapter methods that the new dbt-adapter-test suite can use for testing. ([#2492](https://github.com/fishtown-analytics/dbt/issues/2492), [#2721](https://github.com/fishtown-analytics/dbt/pull/2721))
+- It is now an error to attempt installing `dbt` with a Python version less than 3.6. (resolves [#2347](https://github.com/fishtown-analytics/dbt/issues/2347))
 - Check for Postgres relation names longer than 63 and throw exception. ([#2197](https://github.com/fishtown-analytics/dbt/issues/2197))
 
 
```

Comment:
Can you add a Contributors section (similar to the ones below) and include your GitHub handle + a link to this PR? We want to give you credit!

Reply:
Certainly! See 81bf3da